### ISSUE TITLE
Doesn't seem to work on neovim

### ISSUE THREAD

**Issue by simonweil** [rhysd/committia.vim#22]
Hi,

I'm trying to get it working on neovim but it just hangs.

This is my init.vim:
```Viml
call plug#begin('~/.vim/plugged')
Plug 'rhysd/committia.vim'
call plug#end()
```

Thanks in advance.

**Comment by rhysd** [comment]
Could you give me more information?

1. GUI/CLI
2. OS
3. neovim version
4. when neovim hanging

**Comment by simonweil** [comment]
Here:

1. TUI in iTerm2
2. OSX El Capitan
3. built by brew a week or 2 ago
```
± nvim --version
NVIM 0.1.2-dev
Build type: Dev
Compilation: /usr/local/Library/ENV/4.3/clang -Wconversion -O2 -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wvla -fstack-protector-strong -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -DHAVE_CONFIG_H -I/tmp/neovim20160204-23200-5z459k/build/config -I/tmp/neovim20160204-23200-5z459k/src -I/tmp/neovim20160204-23200-5z459k/deps-build/usr/include -I/tmp/neovim20160204-23200-5z459k/deps-build/usr/include -I/tmp/neovim20160204-23200-5z459k/deps-build/usr/include/luajit-2.0 -I/tmp/neovim20160204-23200-5z459k/deps-build/usr/include -I/tmp/neovim20160204-23200-5z459k/deps-build/usr/include -I/tmp/neovim20160204-23200-5z459k/deps-build/usr/include -I/tmp/neovim20160204-23200-5z459k/deps-build/usr/include -I/usr/local/opt/gettext/include -I/usr/include -I/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include -I/tmp/neovim20160204-23200-5z459k/build/src/nvim/auto -I/tmp/neovim20160204-23200-5z459k/build/include
Compiled by simonweil@Simons-MacBook-Pro.local

Optional features included (+) or not (-): +acl   +iconv    +jemalloc
For differences from Vim, see :help vim-differences

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/Cellar/neovim/HEAD/share/nvim"
```
4. writing `git commit`
vim opens and freezes.
[![asciicast](https://asciinema.org/a/71yd8arelo6uu9yqs7wdon5k5.png)](https://asciinema.org/a/71yd8arelo6uu9yqs7wdon5k5)

**Comment by simonweil** [comment]
After all this I found the problem...
In `committia.vim/autoload/committia/git.vim:2` add `--no-ext-diff` to the git diff command fixes it as my default diff command uses an external program.

I can open a pull request on Sunday if you like.

**Comment by rhysd** [comment]
Thank you for the information (especially video :smile:).  I'll check this tomorrow.

I didn't consider the case when diff using external program.  So I'll check `nvim` or external diff program, which causes this bug.  If external diff program does,  I'll add `--no-ext-diff` as you kindly investigated.

**Comment by rhysd** [comment]
I tested commitia.vim with `nvim` and it worked well also in my environment.  So I think adding `--no-ext-diff` is good.

**Comment by rhysd** [comment]
I fixed this in 4b2e752.
If you notice that bug is still there, please ping me in this issue.

**Comment by simonweil** [comment]
Works great, thank you :)
Now I'm only missing fugitive integration, but that's a different matter.
